### PR TITLE
Column Default Spacing issue

### PR DESCRIPTION
**Pull request by Surezt22**
1. Fixed default spacing issue in columnset.
2. Removed default value for MaxWidth property from FactTitleDefinition in HostConfig.

### CODE REVIEW

**Comment by Surezt22** [comment]
The default spacing issue between the column items when the separator value is false is fixed.

File: source/reactnative/src/components/containers/column.js [hunk 1]
```diff
@@ -47,10 +47,10 @@ export class Column extends React.Component {
 		return renderedElement;
 	}
 
-    /**
-     * @description This function calculates the Width percentage of the column to be 
-	 * rendered based on the width property from the payload. 
-     */
+	/**
```
Comment (BalajiR) [inline source/reactnative/src/components/containers/column.js]: Correct the Indentation.

File: source/reactnative/src/components/containers/column.js [hunk 2]
```diff
@@ -216,7 +213,9 @@ export class Column extends React.Component {
 					<View style={[containerViewStyle, styles.separatorStyle]}>
 						{this.parsePayload()}
 					</View> :
-					this.parsePayload()}
+					<View style={{marginRight: this.spacing}}>
```
Comment (BalajiR) [inline source/reactnative/src/components/containers/column.js]: This suppose to be `marginLeft` for the column right ? Spacing should add the space left to the column in ColumnSet except the first column.

**Comment by Surezt22** [comment]
Changed the property marginRight to marginLeft as the spacing property controls the amount of spacing between this element and the preceding element.

File: source/reactnative/src/components/containers/column.js (hunk 2 above)
Comment (Surezt22) [reply source/reactnative/src/components/containers/column.js]: fixed

File: source/reactnative/src/components/containers/column.js [hunk 3]
```diff
@@ -204,19 +220,21 @@ export class Column extends React.Component {
...
-			actionComponentProps = this.payload.selectAction;
+			actionComponentProps = this.column.selectAction;
 		}
 
-		return <ContainerWrapper json={this.payload} style={[containerViewStyle]}>
+		return <ContainerWrapper json={this.column} style={[containerViewStyle]}>
 			<ActionComponent {...actionComponentProps}>
 				{separator && this.renderSeparator()}
 				{separator ?
 					<View style={[containerViewStyle, styles.separatorStyle]}>
```
Comment (pragadeeshk) [inline source/reactnative/src/components/containers/column.js]: Question: Shouldn't this separator container wrapped with spacingStyle?

Comment (Surezt22) [reply source/reactnative/src/components/containers/column.js]: The separator  rendering logic takes care of spacing while adding the separator line.

Comment (pragadeeshk) [reply source/reactnative/src/components/containers/column.js]: Makes sense. Thanks

**Pull request merged**